tests/__init__.py

```diff
@@ -137,25 +139,21 @@ def check(
 
     value: Any
     if isinstance(actual, pd.Series):
-        value = actual.iloc[  # pyright: ignore[reportUnknownVariableType]
-            index_to_check_for_type
-        ]
+        value = cast(pd.Series, actual).iloc[index_to_check_for_type]
     elif isinstance(actual, pd.Index):
-        value = actual[  # pyright: ignore[reportUnknownVariableType]
-            index_to_check_for_type
-        ]
+        value = cast(pd.Index, actual)[index_to_check_for_type]
```

Comment:
can you put a comment in here that indicates why the `cast` is needed (probably related to https://github.com/microsoft/pyright/issues/11190)

Reply:
looks like he rejected 11191 and 11190 so delete the 11191 comments